ICS-ICAL Sync/Code.gs

```diff
@@ -108,33 +146,49 @@ function main() {
       if (response[i].includes("LOCATION"))
         currentEvent.location = response[i].split("LOCATION:")[1];
         
-      if (response[i].includes("UID"))
+      if (response[i].includes("UID")){
         currentEvent.id = response[i].split("UID:")[1];
+        feedEventIds.push(currentEvent.id);
+      }
     }
   }
   //----------------------------------------------------------------
   
   //------------------------ Check results -------------------------
   Logger.log("# of events: " + events.length);
-  for (var i = 0; i < events.length; i++){
-    Logger.log("Title: " + events[i].title);
-    Logger.log("Id: " + events[i].id);
-    Logger.log("Description: " + events[i].description);
-    Logger.log("Start: " + events[i].startTime);
-    Logger.log("End: " + events[i].endTime);
-    
-    for (var j = 0; j < events[i].reminderTimes.length; j++)
-      Logger.log("Reminder: " + events[i].reminderTimes[j] + " seconds before");
-    
-    Logger.log("");
-  }
+//  for (var i = 0; i < events.length; i++){
```

Comment:
Why were these lines commented out? A change made during your testing?

Reply:
Sorry yea, i kept having to scroll past all the output when debugging and forgot to flip it back.